docs/UserGuide.md

```diff
@@ -150,556 +157,1347 @@ An example of the final command is displayed below.
-* The index **must be a positive integer** 1, 2, 3, …
+<div markdown="block" class="alert alert-example">
+
+:clipboard: **Example 1: Standard command**
+
+* `delete_supplier 2`<br>
+  Deletes the **second** supplier from the visible contact list
+
+:clipboard: **Example 2: Shortcut command after using `find_supplier`**
 
-Examples:
+* `find_supplier n/John` followed by `delete_s 1`<br>
+  * `find_supplier n/John` lists all suppliers with name `John` inside them
+  * `delete_s 1` deletes the **first** supplier with the name `John` from the visible contact list
 
-* `delete_supplier 2` deletes the second supplier
-* `find_s n/John` followed by `delete_s 3` deletes the 1st supplier in the results of the `find_s` command with name `John`
+</div>
+
+### 2.4.2 Deleting an order: `delete_order`
 
-### Deleting an order: `delete_order`
+Deletes the specified order from the order list.
 
 Syntax: `delete_order INDEX`
 
-* Deletes the order at the specified `INDEX`.
-* The index refers to the index number shown in the displayed order list.
-* The index **must be a positive integer** 1, 2, 3, …
+<div markdown="block" class="alert alert-example">
+
+:clipboard: **Example 1: Standard command**
 
-Examples:
+* `delete_order 2`<br>
+  Deletes the **second** order from the visible order list
 
-* `list` followed by `delete_order 2` deletes the 2nd order in Trackr.
-* `find Cake` followed by `delete_order 1` deletes the 1st order in the results of the `find` command.
+:clipboard: **Example 2: Shortcut command after using `find_order`**
+
+* `find_order on/Cake` followed by `delete_o 1`<br>
+  * `find_order on/Cake` lists all orders with item name `Cake` inside them
+  * `delete_o 1` deletes the **first** order with the item name `Cake` from the visible order list
+
+</div>
 
-### Deleting a task: `delete_task`
+### 2.4.3 Deleting a task: `delete_task`
 
 Deletes the specified task from the task list.
 
 Syntax: `delete_task INDEX`
 
-* Deletes the task at the specified `INDEX`.
-* The index refers to the number shown in the task list displayed.
-* The index **must be a positive integer** 1, 2, 3, …
+<div markdown="block" class="alert alert-example">
 
-Examples:
+:clipboard: **Example 1: Standard command**
+
+* `delete_task 2`<br>
+  Deletes the **second** task from the visible task list
+
+:clipboard: **Example 2: Shortcut command after using `find_task`**
+
+* `find_task n/flour` followed by `delete_t 1`<br>
+  * `find_task n/flour` lists all tasks with task name `flour` inside them
+  * `delete_t 1` deletes the **first** task with the task name `flour` from the visible task list
+
+</div>
 
 * `delete_task 2` deletes the first task
 * `find_t flour` followed by `delete_t 3` deletes the 1st task in the result of the `find_t` command
 
-### Deleting a menu item: `delete_item`
+### 2.4.4 Deleting a menu item: `delete_item`
 
-Deletes the specified item from the menu.
+Deletes the specified menu item from the menu.
```

Comment:
Right now, an additional behaviour is that when you delete a menu item, it doesnt invalidate pre-exisiting orders. Might be good to specify this